model-optimizer/mo/main.py

```diff
@@ -96,26 +97,38 @@ def print_argv(argv: argparse.Namespace, is_caffe: bool, is_tf: bool, is_mxnet:
     print('\n'.join(lines), flush=True)
 
 
-def prepare_ir(argv: argparse.Namespace):
+def load_moc_frontends(argv: argparse.Namespace) -> (Optional[FrontEnd], List[str]):
     fem = argv.feManager
-    available_moc_front_ends = []
     moc_front_end = None
 
-    # TODO: in future, check of 'use_legacy_frontend' in argv can be added here (issue 61973)
-    force_use_legacy_frontend = False
-
-    if fem and not force_use_legacy_frontend:
-        available_moc_front_ends = fem.get_available_front_ends()
-        if argv.input_model:
-            if not argv.framework:
-                moc_front_end = fem.load_by_model(argv.input_model)
-                # skip onnx frontend as not fully supported yet (63050)
-                if moc_front_end and moc_front_end.get_name() == "onnx":
-                    moc_front_end = None
-                if moc_front_end:
-                    argv.framework = moc_front_end.get_name()
-            elif argv.framework in available_moc_front_ends:
-                moc_front_end = fem.load_by_framework(argv.framework)
+    # Read user flags:
+    use_legacy_frontend = argv.use_legacy_frontend
+    use_new_frontend = argv.use_new_frontend
+
+    if not fem or use_legacy_frontend:
+        return None, []
```

Comment:
shouldn't we use new front-end for paddle in case `use_legacy_frontend` is `True`? Because this is only front-end for paddle

Reply:
@postrational 
Hi Michal, I see that this comment is not addressed. Or are we not going to invoke pdpd front-end in case legacy option? Please comment on this.

Reply:
I added an error message for the user.  When speficifying both options `--framework=paddle` and `--use_legacy_api`, they would get the message:
```
Framework paddle is not a valid target when using the --use_legacy_frontend flag.
```

Reply:
Changed the error message to:
```
[ ERROR ]  It is not possible to use the --use_legacy_frontend flag when processing a PaddlePaddle model.
```